### PR TITLE
[release-1.17] Automated branch step4

### PR DESCRIPTION
**Pull request by AndreaM12345**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: common/Makefile.common.mk [hunk 1]
```diff
@@ -91,7 +91,7 @@ mirror-licenses: mod-download-go
 	@license-lint --mirror
 
 TMP := $(shell mktemp -d -u)
-UPDATE_BRANCH ?= "master"
+UPDATE_BRANCH ?= "release-1.17"
```
Comment (ericvn) [inline common/Makefile.common.mk:94]: This doesn't need to be in this one, but it's OK.

Comment (ericvn) [reply common/Makefile.common.mk:94]: Because this wasn't merged before Step 5 was done, I have the Step 5 stuff in the other PR to fix this up.

**Review by johnma14 — APPROVED** [review]

**Comment by ericvn** [comment]
I will merge this PR for the RM since it's to the 1.17 branch which is rightfully controlled by them and this is just setting up that permission.

**Pull request merged**